### ISSUE TITLE
[Bug]: Small Nautilus icons in GNOME 43 with High DPI scale factor

### ISSUE THREAD

**Issue by electrickite** [nextcloud/desktop#5133]
### ⚠️ Before submitting, please verify the following: ⚠️

- [X] This is a **bug**, not a question or a configuration issue.
- [X] This issue is **not** already reported on Github (I've searched it).
- [X] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

Running the Nextcloud desktop client on Arch Linux under GNOME 43 with a high DPI monitor and 175% scaling factor. The sync icons in the Nautilus file manager are very small. (see attached screenshot)

![nextcloud-nautilus-icons](https://user-images.githubusercontent.com/3449824/200069017-452988c7-d2a3-4d7f-811f-2c7b8287bf87.png)

[nextcloud-debug.zip](https://github.com/nextcloud/desktop/files/9941788/nextcloud-debug.zip)

### Steps to reproduce

1. Install the nextcloud-client and python-nautilus packages on Arch Linux running GNOME 43.
2. On a high DPI display, set the display scaling factor to 175% in GNOME Settings
3. Run Nextcloud and sync
4. Open the GNOME file manager (nautilus) and navigate to the Nextcloud folder
5. Observe small sync icons on files and folders

### Expected behavior

Sync icons are a reasonable size

### Which files are affected by this bug

N/A

### Operating system

Linux

### Which version of the operating system you are running.

Arch Linux

### Package

Distro package manager

### Nextcloud Server version

25.0.1

### Nextcloud Desktop Client version

1.6.1

### Is this bug present after an update or on a fresh install?

Fresh desktop client install

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### Are you using an external user-backend?

- [X] Default internal user-backend
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

**Comment by MrBn100ful** [comment]
Hi,
I have the same problem, but my scale factor is at 100%

OS: Fedora 37
DE: Gnome 43.0

Nextcloud client: 3.6.1
Nextcloud server: 24.0.7
UI language: French


![image](https://user-images.githubusercontent.com/9916074/201609392-0cfdc4cb-915a-45b9-a610-2117603f28a2.png)

**Comment by JacksonBarker** [comment]
I also experience this issue, hope this insight is helpful to someone.

Fedora 37
GNOME 43.1/Nautilus 43
Wayland

I have two monitors. My primary monitor is scaled to 1920x1080@100%, and the other is scaled to 2256x1504@%150.

Here are my findings:
- Changing scaling settings does not affect sync status icon size.
- The icons are also noticeably smaller on my second monitor (2256x1504@%150).
- Changing the size of the thumbnail icons in nautilus does not seem to change the size of the sync status icons.
- The sync status icons only appear to be a reasonable size on my primary monitor (1920x1080@100%) at the lowest possible thumbnail icon size in nautilus.
- This is not an issue with nautilus 43, icons representing files which are symlinks, shared over network, etc. do not have the same issue

If I find the time to look into the native scaling implementation used for symlink icons and such, I will, But, if anyone else decides to look into this issue, that's where I would start.

![image](https://user-images.githubusercontent.com/57155457/204322695-6742e706-bbcf-44bc-b37f-1a181314eb1e.png)

**Comment by Luo-En** [comment]
I don't think this is a nextcloud issue as I have the same issue with other overlay icons like disks. It looks more like a gnome 43/nautilus 43 bug. I didn't have these issues before updating to gnome 43.

![image](https://user-images.githubusercontent.com/117344815/205877103-73aeada1-345a-40fa-a0fd-435504404f0c.png)

**Comment by Jerome-Herbinet** [comment]
Same here. 
Manjaro with Gnome 43.2
![scr_nc_small_icons](https://user-images.githubusercontent.com/33763786/211394267-ed3e2fd6-a813-4978-9865-cddcea3184af.png)

**Comment by phanky5** [comment]
Same issue on Fedora 38.

**Comment by doppelhelix** [comment]
This is the result of icons generated out of `shell_integration/icons/<size>/oC_xxx.png`
Using icons from `shell_integration/icons/nopadding` will fix this issue

Comparison:
`shell_integration/icons/nopadding/ok.iconset/icon_16x16.png`![](https://raw.githubusercontent.com/nextcloud/desktop/4714e014774f40dc167ad7bd476299535c5057e4/shell_integration/icons/16x16/oC_ok.png)
`shell_integration/icons/16x16/oC_ok.png`![](https://raw.githubusercontent.com/nextcloud/desktop/4714e014774f40dc167ad7bd476299535c5057e4/shell_integration/icons/nopadding/ok.iconset/icon_16x16.png)

**Comment by Jerome-Herbinet** [comment]
@nextcloud/designers

**Comment by szaimen** [comment]
Not sure what designers should do here? Of coures theicons should be well visible...

**Comment by doppelhelix** [comment]
> Not sure what designers should do here?

Actually nothing :)
In this particular case the designers job is done. The icons are already present at `shell_integration/icons/nopadding`. The task at hand is to figure out how to use those instead of the padded ones.

**Comment by mgallien** [comment]
closing as this is a duplicate of #5133
